src/components/Gallery/amp/Gallery.js

```diff
@@ -0,0 +1,39 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Helmet } from 'react-helmet';
+import GalleryContainer from './GalleryContainer';
+
+const Gallery = (props) => {
+  const {
+    width,
+    height,
+    children,
+  } = props;
+
+  const integerWidth = Number.parseInt(width, 10);
+  const integerHeight = Number.parseInt(height, 10);
+
+  return (
+    <Fragment>
+      <Helmet>
```

Comment:
I would suggest to write a utility function to inject script.
`helmet` is almost 7kb gzipped.
https://bundlephobia.com/result?p=react-helmet@5.2.0

Reply:
You're absolutely right, but in the case of AMP, you only do server side rendering, and never execute client side JS, so the `helmet` overhead never reaches the client :)